app/controllers/application_controller.rb

```diff
@@ -13,4 +13,8 @@ def get_alert
 	    @alert = Alert.find_by(published: true)
 	end
 
+  def get_highlights
```

Comment:
Are highlights going to be on every? Trying to understand when this is in the application controller?

Reply:
The homepage isn't a model and doesn't have a controller other than application so that's where I put it.